src/App.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { Button, FormControl, Input } from "@material-ui/core";
+import Service from "./services/getSimilar";
+const service = new Service();
+
+function App() {
+  
+  const [search, setSearch] = useState("");
+  const [result, setResult] = useState();
+
+  const handleChange = ({ target }) => setSearch(target.value);
+
+  const handleSubmit = () => service.getSimilar(search).then(elm => setResult(elm)).catch(err => console.log(err))
+  
+
+  const classes = useStyles();
+
+  return (
+    <div className="App">
+      <form className={classes.root}>
+        <FormControl fullWidth margin="normal">
+          <Input
+            id="Search"
+            label="Search"
+            value={search}
+            onChange={handleChange}
+          />
+        </FormControl>
+        <Button
+          variant="contained"
+          color="secondary"
+          onClick={() => handleSubmit()}
+        >
+          Send
+        </Button>
+      </form>
```

Comment:
you could make this whole thing a separate component and pass down the funstions as props while still keeping the state in the parent component

Reply:
True! Due to the API problems, I did not have time to structure well. But I was asking to be a component